index.ts

```diff
@@ -1 +1,2 @@
-export * from "./src/ExampleComponent";
+// export * from "./src/ExampleComponent";
```

Comment:
I think let's remove the `ExampleComponent` completely now we've got a real version of the banner. What do you reckon?

Reply:
Yes, makes sense to me.